spanner/api/SpannerTest/Tests.cs

```diff
@@ -217,22 +217,23 @@ async void InitializeInstance()
                 string name = $"projects/{_fixture.ProjectId}/instances/{_fixture.InstanceId}";

```

Comment:
I couldn't make this async unfortunately because the compiler complains that we're calling await inside a lock (line 136). But I've now made it synchronous and everything works.